### PR TITLE
add content from old website

### PR DESCRIPTION
**Pull request by mariangz**
### References
-  ⚠️  obsoleted / replaced **we don't have to copy these pages**.  ~(I decided to skip them, after doing a few, because I wasn't sure if we had to)~.
- ✅  copied 
- ✔️ was already moved to the new website (not by me)
- 🔗 external link

### Manuals and Documentation for Users and Administrators
image
image

### Documentation for Developers `/docs`
Maybe we find a way to merge the external and internal categories

image
Screenshot 2024-01-25 at 18 00 34

### Publications  ✅
Screenshot 2024-01-24 at 19 07 54
Screenshot 2024-01-24 at 19 09 10

### Videos for Users and Administrators ✅
Screenshot 2024-01-24 at 19 18 02
Screenshot 2024-01-24 at 19 19 00

### The systemd for Administrators Blog Series ✅
Screenshot 2024-01-24 at 19 19 56
Screenshot 2024-01-24 at 19 20 20

### The systemd for Developers Series ✅
Screenshot 2024-01-24 at 19 37 38
Screenshot 2024-01-24 at 19 39 13

### Related Packages ✅
Screenshot 2024-01-24 at 19 40 31
Screenshot 2024-01-24 at 19 40 58

### CODE REVIEW

**Comment by mariangz** [comment]
links pointing to `freedesktop` still need to be updated.

**Comment by hulkoba** [comment]
# PR description
## Migrate the content of the old website to the new one

This PR moves the content from the old website to this project. 
The categories have been adopted as they are. But we recognized that you've created new categories. So maybe you want to restructure some articles.

This PR uses Jekyll collections to structure the docs. Each markdown file is placed in a directory, named like the category.

Note that the review and editing of the content will happen in a separate PR.
Also note that this PR excludes manpages, as these are addressed in a different milestone.

### Screenshots
Screenshot from 2024-01-30 13-28-05
Screenshot from 2024-01-30 13-28-12
Screenshot from 2024-01-30 13-28-20
Screenshot from 2024-01-30 13-28-30
Screenshot from 2024-01-30 13-28-38

File: docs/_userdocs/DEBUGGING.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
+---
+title: Diagnosing Boot Problems
```
Comment (mariangz) [inline docs/_userdocs/DEBUGGING.md:2]:
```suggestion
title: Debugging Boot Problems
```

File: docs/_userdocs/INCOMPATIBILITIES.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: Compatibility with SysV
```
Comment (mariangz) [inline docs/_userdocs/INCOMPATIBILITIES.md:2]:
```suggestion
title: Incompatibilities with SysV/LSB
```

File: docs/_config.yml
Comment (mariangz) [inline docs/_config.yml:1]: are you going to update this right @ayhamthemayhem ?

**Review by mariangz — COMMENTED** [review]
I left some comments

File: docs/_userdocs/DEBUGGING.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+If your machine gets stuck during boot, first check if the hang happens before or after control passes to systemd.
+
+Try to boot without `rhgb` and `quiet` on the kernel command line. If you see some messages like these:
+
+* Welcome to Fedora _VERSION_ (_codename_)!"
+* Starting _name_...
+* \[ OK \] Started _name_.
+
+then systemd is running. (See an actual [screenshot](f17boot.png).)
```
Comment (mariangz) [inline docs/_userdocs/DEBUGGING.md:18]: it doesn't work the image

File: docs/_userdocs/INCOMPATIBILITIES.md (hunk 2 above)
Comment (hulkoba) [reply docs/_userdocs/INCOMPATIBILITIES.md:2]: The title here is the original https://www.freedesktop.org/wiki/Software/systemd/Incompatibilities/

File: docs/_userdocs/DEBUGGING.md (hunk 1 above)
Comment (hulkoba) [reply docs/_userdocs/DEBUGGING.md:2]: Diagnosing Boot Problems is the original title https://freedesktop.org/wiki/Software/systemd/Debugging/

File: docs/_userdocs/DEBUGGING.md (hunk 3 above)
Comment (hulkoba) [reply docs/_userdocs/DEBUGGING.md:18]: yes, this is for M3

File: docs/_userdocs/INCOMPATIBILITIES.md (hunk 2 above)
Comment (mariangz) [reply docs/_userdocs/INCOMPATIBILITIES.md:2]: but in the index is different, so I wasn't sure what's the original and right one. I thought it should be like the current index page

Comment (hulkoba) [reply docs/_userdocs/INCOMPATIBILITIES.md:2]: We might wanna ask :)

**Pull request merged**